compiler/testData/psi/NewlinesInParentheses.txt

```diff
@@ -242,19 +242,23 @@ KtFile: NewlinesInParentheses.kt
           WHEN_CONDITION_IS_PATTERN
             PsiElement(is)('is')
             PsiWhiteSpace(' ')
-            TYPE_REFERENCE
-              USER_TYPE
-                REFERENCE_EXPRESSION
-                  PsiElement(IDENTIFIER)('T')
-                PsiWhiteSpace('\n    ')
-                TYPE_ARGUMENT_LIST
-                  PsiElement(LT)('<')
-                  TYPE_PROJECTION
+            PATTERN
```

Comment:
Changing PSI for existing files can be dangerous. For example, some IDE actions may cease to operate after it. We should either consider keeping PSI for old code or check attentively that IDE tests (like IntentionTestGenerated, InspectionTestGenerated, LocalInspectionTestGenerated, QuickFixTestGenerated) still pass.